diff --git a/arch/um/Kconfig b/arch/um/Kconfig
index d9541d13d..99dd41673 100644
--- a/arch/um/Kconfig
+++ b/arch/um/Kconfig
@@ -51,7 +51,7 @@ config MMU
 	default y
 
 config UML_DMA_EMULATION
-	bool
+	bool "UML DMA emulation"
 
 config NO_IOMEM
 	bool "disable IOMEM" if EXPERT
@@ -59,7 +59,7 @@ config NO_IOMEM
 	default y
 
 config UML_IOMEM_EMULATION
-	bool
+	bool "UML IOMEM emulation"
 	select INDIRECT_IOMEM
 	select GENERIC_PCI_IOMAP
 
